.github/workflows/ci.yml

```diff
@@ -40,7 +41,7 @@ jobs:
         with:
           bundler-cache: true
       - name: Lint changed files
-        run: bin/rubocop-ci ${{ env.TARGET_BRANCH || env.PREV_SHA }} ${{ env.IS_MERGE }}
+        run: bin/rubocop-ci ${{ env.TARGET_BRANCH || env.PREV_SHA || env.FALLBACK_BRANCH }} ${{ env.IS_MERGE }}
```

Comment:
What if we have a different env (not `IS_MERGE`) when it's triggered manually and then we can set 'develop' as the branch to use in that case?